### PR TITLE
Use `strong` element to semantically denote visually emphasised content

### PR DESCRIPTION
**Pull request by t3chguy**
For https://github.com/element-hq/customer-retainer/issues/217

### CODE REVIEW

File: res/css/views/dialogs/security/_AccessSecretStorageDialog.pcss [hunk 1]
```diff
@@ -117,6 +117,7 @@ limitations under the License.
                 .mx_AccessSecretStorageDialog_reset {
                     position: relative;
                     padding-inline-start: $spacingStart;
+                    font-weight: inherit;
```
Comment (richvdh) [inline res/css/views/dialogs/security/_AccessSecretStorageDialog.pcss:120]: Why is this needed? What happens without it?

**Review by richvdh — APPROVED** [review]

File: res/css/views/dialogs/security/_AccessSecretStorageDialog.pcss (hunk 1 above)
Comment (t3chguy) [reply res/css/views/dialogs/security/_AccessSecretStorageDialog.pcss:120]: The text goes bold, and in this location it makes no sense as we have iconography to back up the colour emphasis. Other sites use merely colour which is not sufficient for visual a11y

Comment (richvdh) [reply res/css/views/dialogs/security/_AccessSecretStorageDialog.pcss:120]: ok, it might be nice to include a comment to say that?
